### PR TITLE
Om feature display error page

### LINKED ISSUE #51: Display error page
Display Error page

 - when the user visits a `route` that doesn't exist the error page should display

### PR DESCRIPTION
**Pull request by Atos20**
### What does this PR do?
- It creates an Error component
- It styles the Error component
- It adds missing paths to components in `App.tsx`
### Where should the reviewer start?
- `src/components/Error/Error.css`
- `src/components/Error/Error.tsx`
- `src/components/App/App.tsx`

### How should this be manually tested?
- run npm start
- write a nonexisting path to the web browser, e.g `http://localhost:3001/nonexistingpath` and the error page should be visible

### What are the relevant tickets?
closes #51 
### Screenshots (if appropriate)
Screen Shot 2021-01-08 at 9 27 20 PM

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -36,17 +36,17 @@ const App:React.FC = () =>{
...
           <Route exact path="/question" component={()=><Question 
             currentQuestion={currentQuestion || "Loading"} updateAllAnswers={updateAllAnswers}/>}/>
-        <Route exact path="/generate_report" component={GenerateReport}/>
-        <Route component={Error}/>
-        <GenerateReport/>
-        <Report />
+          <Route exact path="/generate_report" component={GenerateReport}/>
+          <Route exact path="/submit" GenerateReport/>
+          <Route exact path="/report" component={Report} />
+          <Route path='/*' component={Error}/>
```
Comment (timkeresey) [inline src/components/App/App.tsx:49]: Wouldn't we leave this path declaration off of Error?

**Review by timkeresey — COMMENTED** [review]
Great! If we have time we can maybe add a small animation to this page.

File: src/components/App/App.tsx (hunk 1 above)
Comment (BlakeDonn) [reply src/components/App/App.tsx:49]: I think either way we works!

**Comment by BlakeDonn** [comment]
looks great!

**Pull request merged**